spec/requests/admin/users_spec.rb

```diff
@@ -216,4 +216,89 @@
       end
     end
   end
+
+  describe "blocked user indicator" do
+    context "when user is not blocked" do
+      it "does not show blocked user indicator" do
```

Comment:
I forget how much setup/teardown overhead there is per e2e test, but I wonder if it's worth combining some of these for performance (i.e. doing db operations and then reloading the page to test multiple things in the same spec)